x-pack/plugins/ml/common/types/anomalies.ts

```diff
@@ -36,3 +36,5 @@ export interface AnomalyRecordDoc {
   // TODO provide the causes resource interface.
   causes?: any[];
 }
+
+export { AnomaliesTableRecord } from '../../server/models/results_service/build_anomaly_table_items';
```

Comment:
rather than exporting out an imported interface, `AnomaliesTableRecord` should be moved to this file and imported into the server.

Reply:
took care of it.